services/brig/src/Brig/API/User.hs

```diff
@@ -1264,7 +1264,7 @@ ensureAccountDeleted uid = do
       conCount <- countConnections localUid [(minBound @Relation) .. maxBound]
       cookies <- listCookies uid []
 
-      if (not . null) probs
+      if notNull probs
```

Comment:
there are two more below, now you have to make it consistent.  this is me, promising to be less nit-picky now...  m-)

Reply:
Well, consistency is a good thing. (I've just overseen the other `not.null`s due to the small excerpt in the comment window.) :smile_cat: